packages/mandelbrot/assets/js/components/preview.js

```diff
@@ -60,6 +65,28 @@ export default class Preview {
         });
     }
 
+    _updateIframeRefs() {
+        this._iframe = this._el.find('[data-role="window"]');
+        this._previewIframeWindow = this._iframe ? this._iframe.get(0).contentWindow : null;
+        this._previewSize = $('[data-role="preview-size"]');
```

Comment:
Not sure about the global query here - I think everywhere else element queries are scoped to `this._el`. Maybe it would be better to query this element in Pen class and pass it into the Preview constructor?

Reply:
Like this?

Reply:
Yup, looks good now 👍